deploy/olm-catalog/bundle/manifests/migration.openshift.io_migclusters.yaml

```diff
@@ -1,3 +1,5 @@
+
```

Comment:
Please remove these 2 lines. They will (and have repeatedly in the past with our and velero CRD) break OLM. OLM process the empty doc, stops and we never get the CRD installed, which breaks everything.

Reply:
ACK, I'm a bit surprised this causes problems, but will take it out. This gets auto-generated from controller-tools. Do you remember if this was always a problem or if it started recently? We updated controller-tools a few months back.

Reply:
Updated @jmontleon

Reply:
@djwhatle "This gets auto-generated from controller-tools" -- yes. In fact when I pull the velero CRD definitions into mig-operator I have to remove the extra lines from those in the upstream CRD definitions for similar reasons.